### PR TITLE
[Email] new delete email flow

### PR DESCRIPTION
**Pull request by KayMKM**
### Purpose
*What is the context of this pull request? Why is it being done?*
Close: https://github.com/microsoft/botframework-skills/issues/79

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*
image
image
image

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

File: skills/csharp/emailskill/Dialogs/EmailSkillDialogBase.cs [hunk 1]
```diff
@@ -1270,13 +1267,62 @@ protected async Task ClearAllState(WaterfallStepContext sc)
         protected async Task DigestFocusEmailAsync(WaterfallStepContext sc)
         {
             var state = await EmailStateAccessor.GetAsync(sc.Context);
+            sc.Context.Activity.Properties.TryGetValue("OriginText", out var content);
```
Comment (xieofxie) [inline skills/csharp/emailskill/Dialogs/EmailSkillDialogBase.cs]: what is OriginText?

File: skills/csharp/tests/emailskill.tests/Flow/DeleteEmailFlowTests.cs [hunk 2]
```diff
@@ -31,7 +31,6 @@ await this.GetTestFlow()
                 .Send(BaseTestUtterances.FirstOne)
                 .AssertReply(this.DeleteConfirm())
                 .Send(GeneralTestUtterances.No)
-                .AssertReplyOneOf(this.CancelResponses())
                 .StartTestAsync();
         }
```
Comment (xieofxie) [inline skills/csharp/tests/emailskill.tests/Flow/DeleteEmailFlowTests.cs]: Could we add some new tests? Since it is 'new delete email flow'..

Comment (KayMKM) [reply skills/csharp/tests/emailskill.tests/Flow/DeleteEmailFlowTests.cs]: updated test

File: skills/csharp/emailskill/Dialogs/EmailSkillDialogBase.cs (hunk 1 above)
Comment (KayMKM) [reply skills/csharp/emailskill/Dialogs/EmailSkillDialogBase.cs]: Seems not be used now.. Removed all OriginText

File: skills/csharp/emailskill/Responses/Shared/LgIdCollection.t4 [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+// Licensed under the MIT License.
+
+using Microsoft.Bot.Solutions.Responses;
+
+namespace <#= namespaceName #>
+{
+    /// <summary>
+    /// Contains bot responses.
+    /// </summary>
+    public class <#= className #> : IResponseIdCollection
```
Comment (xieofxie) [inline skills/csharp/emailskill/Responses/Shared/LgIdCollection.t4]: this conflicts with https://github.com/microsoft/botframework-skills/pull/117/files

Comment (KayMKM) [reply skills/csharp/emailskill/Responses/Shared/LgIdCollection.t4]: fixed

File: skills/csharp/emailskill/Responses/ShowEmail/ShowEmailTexts.it-it.lg [hunk 4]
```diff
@@ -13,3 +13,6 @@
 # ReadOutMessage.Speak(emailDetailsWithContent)
 - Ecco i dettagli dell'e-mail: ${emailDetailsWithoutContent}
 
+# ActionPrompt.Text
+- To take an action on an email, first select it, then you can delete, forward, or reply.
```
Comment (lzc850612) [inline skills/csharp/emailskill/Responses/ShowEmail/ShowEmailTexts.it-it.lg]: can you use Google translate to translate these to respective languages?

Comment (KayMKM) [reply skills/csharp/emailskill/Responses/ShowEmail/ShowEmailTexts.it-it.lg]: sure. fixed

**Pull request merged**